service_container/autowiring.rst

```diff
@@ -520,7 +520,7 @@ the injection::
 
             $services->set(TwitterClient::class)
                 // the Rot13Transformer will be passed as the $transformer argument
-                ->autowire()
+                ->autowire();
```

Comment:
Can we move this semicolon to a new line below line 528? I want people to easily uncomment the lines below